packages/contracts/tasks/runner/vote.ts

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable no-console */
+import { task, types } from "hardhat/config";
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { Signer } from "ethers";
+import type { BaseContract } from "ethers";
+import type { MACI } from "../../typechain-types";
+import { PrivKey, PubKey } from "@maci-protocol/domainobjs";
+import {
+  info, 
+  success,
+  logGreen, 
+  logYellow,
+  logRed
+} from "../../ts/logger";
+import { getPollParams, publish, getJoinedUserData } from "@maci-protocol/sdk";
```

Comment:
You can't use `@maci-protocol/sdk` in contracts because sdk package imports contracts.
Right now there is a cycle dependency.